### PR TITLE
Feature/events carousel

### PR DESCRIPTION
**Pull request by caritonaranjo**
Create the component Eventos Destacados.

trello ticket: https://trello.com/c/lIfqXcPA/19-mobile-eventos-destacados-8
design : https://app.zeplin.io/project/5f00e6fda64d6e11dca0753a/screen/5f00e7cd0312ce95b07caa0d

1. create the js file.
2. create the sass file.
3. create the carousel in the Dom file and the styles.
4. animated the carousel.
5 fix the bugs and errors in the sass file.

### CODE REVIEW

File: src/js/components/events-carousel.js [hunk 1]
```diff
@@ -0,0 +1,78 @@
+import { newText, newContainer, newMultimedia } from './helpers'
```
Comment (camilomejiag) [inline src/js/components/events-carousel.js:1]: Please separate the `imports` section from the rest of the component's code with an extra space

File: src/js/components/events-carousel.js [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+const createCards = (cardList) => {
+  const cardCenter = cardList.map((card, index) => {
+    const liClasses = index === 0 ? ['carousel-card', 'carousel-card--active'] : ['carousel-card'];
+    const cardImage = newMultimedia('img', card, ['carousel-card__img']);
+    const cardItem = newContainer('li', [cardImage], liClasses);
+
+    return cardItem;
+  });
+  return cardCenter;
+};
```
Comment (camilomejiag) [inline src/js/components/events-carousel.js:16]: Add an extra space between functions

File: src/js/components/events-carousel.js [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+const clickLeft = () => {
+  const buttonLeft = document.querySelector('.carousel__button--left');
+  const buttonRigth = document.querySelector('.carousel__button--right');
+  if (buttonLeft.classList.contains('carousel__button--disabled')) return;
+
+  buttonRigth.classList.remove('carousel__button--disabled');
+  cardList[currentCard].classList.remove('carousel-card--active');
+  currentCard--;
+  cardList[currentCard].classList.add('carousel-card--active');
+  if (currentCard === 0){
```
Comment (camilomejiag) [inline src/js/components/events-carousel.js]: Same here with the negation (`!`)

File: src/js/components/events-carousel.js [hunk 4]
```diff
@@ -0,0 +1,78 @@
+import { newText, newContainer, newMultimedia } from './helpers'
+let currentCard = 0;
+let cardList;
+
+const createCards = (cardList) => {
+  const cardCenter = cardList.map((card, index) => {
+    const liClasses = index === 0 ? ['carousel-card', 'carousel-card--active'] : ['carousel-card'];
```
Comment (camilomejiag) [inline src/js/components/events-carousel.js]: `index === 0` can be replaced with a negation (`!`), no? It looks fancier :D

**Review by Diegopariza — APPROVED** [review]

File: src/data/data.js [hunk 5]
```diff
@@ -78,19 +78,21 @@ const pageData = {
...
         images: [
           {
-            src: '',
+            src: 'event-1.jpg',
             alt: 'Piscina',
           },
           {
-            src: '',
+            src: 'event-2.png',
+            alt: 'Catalita',
```
Comment (mfbarrera) [inline src/data/data.js:92]: Catalina esta mal escrio

File: src/data/data.js [hunk 6]
```diff
@@ -78,19 +78,21 @@ const pageData = {
...
+          },
+          {
+            src: 'event-3.png',
             alt: 'Catalita',
```
Comment (mfbarrera) [inline src/data/data.js:96]: lo mismo aqui

File: src/data/data.js (hunk 5 above)
Comment (caritonaranjo) [reply src/data/data.js:92]: esas imágenes son solo de prueba, no si si ese contenido es el real, entonces por eso no lo modifique.

**Review by camilomejiag — APPROVED** [review]

**Pull request merged**